mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/query/SortConverter.java

```diff
@@ -55,7 +55,7 @@ private static FieldSortBuilder getSortClause(SearchQuery.Sort.SortClause clause
             case Size :
                 return SortBuilders.fieldSort(JsonMessageConstants.SIZE);
             case SentDate :
-                return SortBuilders.fieldSort(JsonMessageConstants.SENT_DATE);
+                return SortBuilders.fieldSort(JsonMessageConstants.DATE);
```

Comment:
We have to make ES match what was written for send date.

But that's what ES does, as far as I remember: use the Date header and defaults to internal date.